controllers/scaledjob_controller.go

```diff
@@ -29,12 +34,13 @@ type ScaledJobReconciler struct {
 	Log               logr.Logger
 	Scheme            *runtime.Scheme
 	GlobalHTTPTimeout time.Duration
+	Recorder          record.EventRecorder
 	scaleHandler      scaling.ScaleHandler
 }
 
 // SetupWithManager initializes the ScaledJobReconciler instance and starts a new controller managed by the passed Manager instance.
 func (r *ScaledJobReconciler) SetupWithManager(mgr ctrl.Manager) error {
-	r.scaleHandler = scaling.NewScaleHandler(mgr.GetClient(), nil, mgr.GetScheme(), r.GlobalHTTPTimeout)
+	r.scaleHandler = scaling.NewScaleHandler(mgr.GetClient(), nil, mgr.GetScheme(), r.GlobalHTTPTimeout, mgr.GetEventRecorderFor("scale-handler"))
```

Comment:
Event recorder for Metrics Adapter is named `keda-metrics-adapter` and this one is named `scale-handler`. For consistency, this one could be maybe named `keda-operator`/ `keda-controller`. WDYT?